docs/csharp/tutorials/intro-to-csharp/interpolated-strings.yml

```diff
@@ -129,9 +129,6 @@ items:
     > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
 
 - content: |
-    You've completed the string interpolation quickstart. 
-    
-    You can continue these quickstarts in your own local development environment. Learn the basics of [local development](local-environment.md) and then pick a quickstart. You can try this same quickstart, move directly to the next quickstart, or start again with the [Numbers in C#](numbers-in-csharp-local.md) quickstart.
-
-    For more information, see the [String interpolation](../../language-reference/tokens/interpolated.md) topic and the [String interpolation in C#](../../tutorials/string-interpolation.md) tutorial.
+    You've completed the string interpolation quickstart. You can click the **Next** link below to start the **Collections introduction to C# interactive tutorial**, or you can visit the [.NET site](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial) to download the .NET Core SDK, create a project on your machine, and keep coding. The "Keep Learning" step brings you back to these tutorials.
```

Comment:
- Hasn't the quickstart been renamed to a tutorial?
- There is ndo **Next** link; the link reads **Collections in C#**-
- **Collections introduction...** --> the title reads **Learn to managed data collections using the generic list type**
- Keep learning step? I just don't see it or know what it is.